packages/kit/src/views/Market/MarketDetailV2/components/InformationTabs/components/Holders/HolderItem.tsx

```
@@ -0,0 +1,88 @@
+  };
+
+  const formatAmount = (amount: string) => {
+    const num = parseFloat(amount);
+    if (num < 0.001) {
+      return num.toExponential(2);
+    }
+    return numberFormat(amount, { formatter: 'balance' });
+  };
+
+  const formatFiatValue = (fiatValue: string) => {
+    const num = parseFloat(fiatValue);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)